### PR TITLE
Change Network Event type to Raise Up Initiative

### PR DESCRIPTION
**Pull request by saabdoli**
Changed an an event type option to more accurately
represent the name of the organization

### CODE REVIEW

File: public/modules/network-events/views/create-network-event.client.view.html [hunk 1]
```diff
@@ -18,6 +18,51 @@
     					<select id="eventType" name="eventType" class="form-control" data-ng-model="networkEvent.eventType">
                             <option value="">-- Select an Event Type -- </option>
     						<option>College 101</option>
+    						<option>Raise Up Initiative</option>
```
Comment (anthonycrumley) [inline public/modules/network-events/views/create-network-event.client.view.html]: Looks like a strange copy and paste thing happened here and it duplicated the code on this view.

Comment (anthonycrumley) [inline public/modules/network-events/views/create-network-event.client.view.html]: I always do these two things:
* Before each commit I do a `git diff` to make sure I didn't make unintended changes.  If the changes are already added then I do a `git diff --cached`.
* When I create a pull request in github, I always look at the `Files changed` tab to make sure that I didn't include unintended changes.

I am sorta paranoid that way.  :)

Comment (saabdoli) [inline public/modules/network-events/views/create-network-event.client.view.html]: OK, thank you for the advice.

**Comment by anthonycrumley** [comment]
:+1:  Thanks for all the work you are doing for Birmingham's kids!!

**Pull request merged**